content/workload-reduction-toolkit/address-workload-issues/feedback-and-marking/give-feedback-effectively-without-creating-an-unmanageable-workload.md

```diff
@@ -0,0 +1,207 @@
+---
+title: Give feedback effectively without creating an unmanageable workload
+layout: /markdown_resource.html.erb
+---
+
+# Give feedback effectively without creating an unmanageable workload
+
+<strong class="govuk-tag">Example</strong>
+
+{inset-text}
+
+## School details
+
+**School name:** Outwood Grange Academies Trust
+
+**Location:** Multiple locations across the north of England and the East Midlands
+
+**Phase:** Primary and secondary (this example is designed for secondary settings)
+
+**Number of pupils:** 25,000 pupils across 44 academies
+
+**Contact details:** Email the academy trust at <enquiries@outwood.com> 
+
+{/inset-text}
+
+<div class="govuk-grid-row dfe-width-container">
+  <div class="govuk-grid-column-full">
+    <div class="info-box">
+      <div class="info-box__corner">
+        <img src="/assets/images/bullseye.svg" alt="Bullseye icon">
+      </div>
+      <h2 class="govuk-heading-m">
+        Impact and outcomes
+      </h2>
+      <p>
+        Outwood Grange Academies Trust has created a culture where teachers give feedback effectively without creating an unmanageable workload. 
+      </p>
+      <p>
+        Thanks to their approach, teachers are innovative in how they provide feedback. They’re empowered to:
+        <ul>
+          <li>give feedback and mark in a way that best suits their subject and pupils</li>
+          <li>use a wide variety of feedback strategies such as verbal feedback and live marking</li>
+        </ul>
+      </p>
+      <p>
+        Teachers are also more selective in what, why and how they mark. They:
+        <ul>
+          <li>focus on what makes a difference in written marking</li>
+          <li>mark for pupils - not leaders</li>
+        </ul>
+      </p>
+      <p>
+      Workload has also reduced and there is a better balance between time spent on planning and marking. Pupils are more actively engaged in checking their own work and responding to feedback.
+      </p>
+    </div>
+  </div>
+</div>
+
+## Background from Executive Principal Lynn James
+
+{inset-text}
+We have reduced the pressures which increase teacher anxiety, ensuring teachers can excel and focus on what makes the biggest difference. 
+
+Our quality assurance process removes graded lesson observations and instead evaluates how well a subject’s curriculum secures pupils’ learning.  
+
+Pupils’ books become a reflection of effective learning, not a check of marking compliance. 
+{/inset-text}
+
+## Our approach to feedback and marking 
+
+We believe that teachers should spend more time planning how to deliver their curriculum than marking as this has a greater impact on pupils’ learning.  
```

Comment:
Proposed re-ordering: 'We believe that teachers should spend less time marking than planning how to deliver their curriculum, as this has a greater impact on pupils' learning'

Reply:
I'm actually going to suggest removing the line as I don't think it's completely relevant if thats ok!